doc/DEPENDENCIES.md

```diff
@@ -101,6 +102,52 @@ The variable should be set to the value of the **boost root directory**, which i
 
 At present, a version >= `1.72.0` is required.
 
+## Udunits
+
+[Udunits](https://www.unidata.ucar.edu/software/udunits) is a C library and associated command line tool for converting between compatible units.  Only the C library is a required dependency, though these frequently get bundled together.
+
+### Setup
+
+The description of the setup is more complicated than most other current dependencies.  However, the required steps should be very easy to perform.
+
+Basically, the library and needed development files can be installed via typical package managing tools.  But, the ngen project build is also be able to automatically obtain and build it, but only when the transitive dependencies are already available.
+
+#### Option 1: Package Manager Install
+
+The Udunits library and development files can be installed via standard OS package management tools.  The ngen project will automatically detect and use these.  Note that both the library itself and development files must be installed, which are typically two separate, similarly named packages.
+
+The ngen build will automatically search for these, so no additional steps are required to configure the build or the environment once they are installed.  Installation can be done using the typical OS package manager, like `yum` or `apt`.  Different distributions like to call the packages different things (e.g., _udunits2-dev_, _udunits2-devel_, _libudunits2-dev_, etc.), so you may need to do some searching.  Also, for CentOS, you'll need [the EPEL repo enabled](https://docs.fedoraproject.org/en-US/epel/)
+
+Packages for both the _library_ and _development files_ are needed.  Package managers usually make this easy, but just make sure to pay attention to it if things do not work as expected.
```

Comment:
Might be worth dropping this link in to help find the correct package names for different distros:
https://pkgs.org/search/?q=udunits

Reply:
Added in `71cac887c60c3e2c3dafc64ca6ccebd150b18d30`.